manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811
+PackageLocale: en-US
+Publisher: Troye Guo

```

```suggestion
Publisher:  App by Troye
```
This must match the entry in control panel